src/vcf/store_parse_policy.cpp

```diff
@@ -161,37 +158,37 @@ namespace ebi
 
     void StoreParsePolicy::handle_body_line(ParsingState const & state) 
     {
+        // Transform the position token into a size_t
+        auto position = static_cast<size_t>(std::stoi(m_line_tokens["POS"][0]));
+
+        // Transform all the quality tokens into floating point numbers
+        float quality;
         try {
-            // Transform the position token into a size_t
-            auto position = static_cast<size_t>(std::stoi(m_line_tokens["POS"][0]));
```

Comment:
oh, tests fail because a '.' is a valid quality. I'll add some code to test it. if quality is '.', are we happy with putting a 0?